### ISSUE TITLE
limit Layout/RedundantLineBreak block nesting

### ISSUE THREAD

**Issue by mockdeep** [rubocop/rubocop#10310]
## Is your feature request related to a problem? Please describe.

Not really a problem, just a code smell.

## Describe the solution you'd like

I really like the `InspectBlocks` setting for the `Layout/RedundantLineBreak` rule, but I would like to be able to limit the number of nested blocks that the linter reports. For example, if there are blocks inside of blocks, then I would want the outer blocks to be wrapped:

```ruby
# bad
respond_to { |format| format.json { render(json: feedbacks) } }

# bad
respond_to do |format|
  format.json do
    render(json: feedbacks)
  end
end

# good
respond_to do |format|
  format.json { render(json: feedbacks) }
end
```

## Describe alternatives you've considered

Well, for now either accepting slightly more confusing single-line blocks, or keeping the rule disabled and having a lot of blocks that could be reasonably wrapped.

**Comment by jonas054** [comment]
My PR was not accepted, so I'm closing this issue.
